server/app/views/admin/programs/ProgramBlockView.java

```diff
@@ -13,22 +14,30 @@
 
 abstract class ProgramBlockView extends BaseHtmlView {
   /** Renders a div with internal/admin program information. */
-  protected final DivTag renderProgramInfo(ProgramDefinition programDefinition) {
-    DivTag programTitle =
-        div(programDefinition.adminName()).withId("program-title").withClasses("text-3xl", "pb-3");
-    DivTag programDescription = div(programDefinition.adminDescription()).withClasses("text-sm");
+  protected final DivTag renderProgramInfo(ProgramDefinition program) {
```

Comment:
There is a "program" concept also; suggestion to make the name shorter with `programDef`